src/api/helpers/map-application-submission-deadline-variables/index.test.ts

```diff
@@ -0,0 +1,75 @@
+import dotenv from 'dotenv';
+import formatDate from '../format-date';
+import mockApplication from '../../test-mocks/mock-application';
+import generateApplicationUrl from '../generate-application-url';
+import mapApplicationSubmissionDeadlineVariables from '.';
+
+dotenv.config();
+
+describe('helpers/map-application-submission-deadline-variables', () => {
+  const application = {
+    owner: {
+      firstName: 'firstName',
+      lastName: 'lastName',
+      email: process.env.GOV_NOTIFY_EMAIL_RECIPIENT_1,
+    },
+    referenceNumber: 12345,
+    submissionDeadline: new Date(),
+    buyer: mockApplication.buyer,
+  };
```

Comment:
Could this come from test mocks instead?

Could also destructure `application.owner` and `application.buyer` and `referenceNumber` as it's repeated a lot in the tests below